### DISCUSSION TITLE
can't schedule more than 7 messages for send-later

### DISCUSSION THREAD

**Discussion by pmamayeri** [Extended-Thunder/send-later#392] [Q&A]
For some reason, after I've scheduled 7 messages for send-later, I can't open the send-later window until some of the previously schedules messages are sent. I'm running thunderbird 91.2.0 (32-bit) and send-later version 9.0.6 on win-10 (64bit)

**Comment by jph-sendlater** [comment]
That is very strange. I don't have a good idea what might cause that right now. Unfortunately, send later development will be very slow and limited to urgent bug fixes for the time being. I just started a new job, and it's taking up a lot of my mental bandwidth right now. I'll look into this when I get a chance.

**Reply by pmamayeri** [reply]
I'm beginning to suspect that it isn't a simple number of items queued, but a possible locking issue ? is there any kind of lock involved when clicking on the "send later" tab to open the "send later" dialog window ? maybe some combination of actions leaves it locked and eventually gets unlocked by other actions including the sending of pre-queued emails.

**Comment by jikamens** [comment]
Is this still an issue?

**Comment by jikamens** [comment]
Closing because no response. If this is still happening [open an issue](https://github.com/Extended-Thunder/send-later/issues/new) please.
